lte/cloud/helm/lte-orc8r/values.yaml

```
@@ -137,3 +137,13 @@ smsd:
       orc8r.io/swagger_spec: "true"
     annotations:
       orc8r.io/obsidian_handlers_path_prefixes: "/magma/v1/lte/:network_id/sms"
+
+nprobe:
+  service:
+    labels:
+      orc8r.io/obsidian_handlers: "true"
+      orc8r.io/swagger_spec: "true"
+    annotations:
+      orc8r.io/obsidian_handlers_path_prefixes: >
+        /magma/v1/lte/:network_id/network_probe/tasks,
+        /magma/v1/lte/:network_id/network_probe/destinations,
```

This routing is based on prefixes, so just one prefix is sufficient
```suggestion
      orc8r.io/obsidian_handlers_path_prefixes: >
        /magma/v1/lte/:network_id/network_probe,
```